mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.erb

```diff
@@ -605,23 +630,6 @@ resource "google_compute_subnetwork" "network-with-private-secondary-ip-ranges"
 `, cnName, subnetworkName)
 }
 
-func testAccComputeSubnetwork_secondaryIpRanges_update4(cnName, subnetworkName string) string {
```

Comment:
I think a recent commit removed the wrong configuration! We want to remove the diff in this file entirely.

Reply:
my mistake! should be good now.